testing/consumer-test/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "consumer-test",
+  "private": true,
+  "version": "1.0.0",
+  "scripts": {
+    "start": "ts-node run.ts"
+  },
+  "dependencies": {
```

Comment:
I believe we should place the other libraries in here as well since this is a typescript version compatibility check. While it seems highly unlikely, it would be agonizing to get a package so close to GA and then have to revisit the code to get it to "work" with the same range of TypeScript versions supported by shipped packages.

@joshgummersall what do you think?

Reply:
Yeah, I'll add them. Good call!